Bugfixes+minor tweaks+altered 2nd Ljublana award

marijn211 opened the pull request:
Italy will only take the Aegean Islands and Crete instead of taking basically all of mainland Greece if Bulgaria is involved in the 2nd ljubljana award, Italy's event fired only rarely because it needs to do a focus that already gives them a wargoal on Greece, I swapped some requirements to make this focus relevant for now as Italy still has only one alternate branch, this is a-historical but -before it happened rarely that Italy got anything here -it was too OP if they did -it was already ahistorical kinda because an immediate war choice would be more accurate I guess
‎
‎
Also puts a state check on Greece's R56 Dodecanese demanding, they cant demand it if they gave their islands to Italy

Also makes a focus in Trotsky's tree boost communism so you aren't stuck on down to 50% support depending on what event decisions you make during the coup (0.10, would seem too high but this is scaled down to 0.02)

Makes Albania less likely to give in to being puppeted by Greece, the last factor comparison was up to 2000:30, now its closer to 600:30 at most and down to 120:60 at least when Albania is at low strength

Locks Britain from able to do Scandinavian intervention right from the start because of R56's different ideology values for Sweden

Allows communist Yugoslavia to cancel (keeps progress ig) coup preparations on a country if that country joins their faction (due to war most likely)(not done by AI, unknown if thats an issue in reality), also shows Turkey can be influenced right away when doing the 2nd balkan influencing focus, but still locks it behind doing the 3 required focuses for the diplomacy focus.

Related optional changes that could later be made: tweak Albania's AI thing with Greece puppeting it even more in favor of independence with more/different factors, the strength factors will almost always be in favor of Greece, reducing variety

marijn211 committed:
Fixed German-related changes

marijn211 committed:
Ownership check

Prevents Greece from demanding the Dodecanese after they had to give them away

marijn211 committed:
Less free land for Italy

marijn211 committed:
Very minor spelling inconsistency

2 different spellings of defense in the same branch

marijn211 committed:
Modified Greek events file

Thinking-waffle reviewed (changes requested):
Don't accept this until after the monthly update, requires testing and marijn proper inclusion into the team.

marijn211 committed:
Tweak-fitting localisation

marijn211 committed:
Swapped tension+More extensive bypass

Moves the tension of the second Ljublana reward to Fate of Greece, this makes more sense with Road to 56's different focus layout

(war checks removed because the focus cancels if Greece is at war)
(bunch of indentation)

marijn211 committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

marijn211 committed:
Trotsky focus adds communism

Puts a communist drift on Trotskyite narrative so you arent stuck with <45% support for communism

marijn211 committed:
Merge pull request #1 from marijn2112/patch-1

Patch 1

marijn211 committed:
Trotsky focus gives communism support

Increased previous value after testing, effect is scaled down to 25% so it was quite little and other effects made the net change negative

marijn211 committed:
War check on "free" land

No more puppeting Albania (which also shouldnt give in in YUGs faction) while at war

marijn211 committed:
More less easy free land

Greece stop stealing my Albania

marijn211 committed:
Stop Scandinavian intervention when not needed

Stops Scandinavian intervention from being possibly taken right at the start of the game because Sweden has some party variety while not even switching ideology

marijn211 committed:
No civil war for faction member

Allow stopping of communist civil war instigation on faction members
(AI wont do, can be made to do so if necessary)

marijn211 committed:
Mistake fix

added a missing OR gate, because it was inside a NOT and it was needed there

marijn211 committed:
Slightly prettier and more readable notation

marijn211 committed:
Bugfix

Tags had not been changed after copy paste of similar focus

marijn211 commented:
Should only need 1 additional change namely changing the vanilla/DLC tree to also use my altered 2nd ljbuljana award

marijn211 committed:
Match r56 tree changes

marijn211 committed:
Unhide Turkish coup possibility

Let the player know this decision exists, moved visible to available

marijn211 committed:
(No) DLC-proofed focus

Now you can use it without La Resistance

marijn211 committed:
Small Ottoman Turkey restructure

Removes some broken/silly bypasses and restructures a bit so you can complete the tree without having to bypass any buff focuses

marijn211 committed:
Update turkey.txt

marijn211 committed:
Copy pasta fix

Fixes a bug result of a bad copy paste

ilkober reviewed (commented):
Unsure about the changes to events/Germany.txt and "new" Greek event
The rest looks fine and has Fantoms seal of approval

marijn211 committed:
Fix kinda

Another case where the player can't choose not to join a faction, so it shouldn't force sideeffects

marijn211 committed:
Consistency

marijn211 committed:
Add new focus to AI strategy
